changelogs/current.yaml

```diff
@@ -180,6 +180,9 @@ bug_fixes:
     Validate http service path_prefix
     :ref:`path_prefix <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.HttpService.path_prefix>`,
     Validate http service path_prefix configuration must start with ``/``.
+- area: admin
+  change: |
+    Fixed missing additional addresses for :ref:`LbEndpoint <envoy_v3_api_field_config.endpoint.v3.LbEndpoint.endpoint>` in config dump.
```

Comment:
Can you please add a link for `additional addresses` (to this field: https://github.com/envoyproxy/envoy/blob/de6b730c3f59e8ee3ee756c60bc826d7ede295e1/api/envoy/config/endpoint/v3/endpoint_components.proto#L60).

Reply:
Done